packages/ui/src/theme/tokens/components/index.ts

```diff
@@ -1,42 +1,87 @@
-import { heading } from './heading';
-import { icon } from './icon';
-import { image } from './image';
-import { link } from './link';
-import { loader } from './loader';
-import { menu } from './menu';
-import { pagination } from './pagination';
-import { phonenumberfield } from './phoneNumberField';
-import { placeholder } from './placeholder';
-import { radio } from './radio';
-import { rating } from './rating';
-import { select } from './select';
-import { selectfield } from './selectField';
-import { sliderfield } from './sliderField';
-import { stepperfield } from './stepperField';
-import { switchfield } from './switchField';
-import { table } from './table';
-import { tabs } from './tabs';
-import { text } from './text';
-import { togglebutton } from './toggleButton';
-import { togglebuttongroup } from './toggleButtonGroup';
+import { PaginationInput } from '@aws-amplify/datastore';
+import { AlertTokens, alert } from './alert';
+import { AuthenticatorTokens, authenticator } from './authenticator';
+import { badge, BadgeTokens } from './badge';
+import { button, ButtonTokens } from './button';
+import { card, CardTokens } from './card';
+import { checkbox, CheckboxTokens } from './checkbox';
+import { checkboxfield, CheckboxFieldTokens } from './checkboxField';
+import { copy, CopyTokens } from './copy';
+import {
+  countrycodeselect,
+  CountryCodeSelectTokens,
+} from './countryCodeSelect';
+import { divider, DividerTokens } from './divider';
+import { expander, ExpanderTokens } from './expander';
+import { field, FieldTokens } from './field';
+import { fieldcontrol, FieldControlTokens } from './fieldControl';
+import { fieldgroup, FieldGroupTokens } from './fieldGroup';
+import { fieldmessages, FieldMessagesTokens } from './fieldMessages';
+import { flex, FlexTokens } from './flex';
+import { heading, HeadingTokens } from './heading';
+import { icon, IconTokens } from './icon';
+import { image, ImageTokens } from './image';
+import { link, LinkTokens } from './link';
+import { loader, LoaderTokens } from './loader';
+import { menu, MenuTokens } from './menu';
+import { pagination, PaginationTokens } from './pagination';
+import { placeholder, PlaceholderTokens } from './placeholder';
+import { radio, RadioTokens } from './radio';
+import { rating, RatingTokens } from './rating';
+import { select, SelectTokens } from './select';
+import { selectfield, SelectFieldTokens } from './selectField';
+import { sliderfield, SliderFieldTokens } from './sliderField';
+import { stepperfield, StepperFieldTokens } from './stepperField';
+import { switchfield, SwitchFieldTokens } from './switchField';
+import { table, TableTokens } from './table';
+import { tabs, TabsTokens } from './tabs';
+import { text, TextTokens } from './text';
+import { togglebutton, ToggleButtonTokens } from './toggleButton';
+import {
+  togglebuttongroup,
+  ToggleButtonGroupTokens,
+} from './toggleButtonGroup';
+
+export interface Components {
```

Comment:
With `Components` already being somewhat of an overloaded term, maybe we can name this `ComponentTokens`?

Reply:
Yes, definitely. Will update